rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/CloudProviderBakeHandler.groovy

```diff
@@ -140,7 +141,21 @@ abstract class CloudProviderBakeHandler {
     def virtualizationSettings = findVirtualizationSettings(region, bakeRequest)
 
     BakeOptions.Selected selectedOptions = new BakeOptions.Selected(baseImage: findBaseImage(bakeRequest))
-    def (imageName, appVersionStr, packagesParameter) = imageNameFactory.deriveImageNameAndAppVersion(bakeRequest, selectedOptions)
+
+    List<String> packageNameList = bakeRequest?.package_name?.tokenize(" ")
+    def appVersionStr
+
+    def osPackages = packageNameList.collect { name ->
+      PackageNameConverter.buildOsPackageName(selectedOptions.baseImage.packageType, name)
+    }
+
+    if (!osPackages.empty) {
```

Comment:
Can typically just rely on Groovy truthiness when operating on a list (http://mrhaki.blogspot.com/2009/08/groovy-goodness-tell-groovy-truth.html). So this could read: `if (osPackages) {`